### ISSUE TITLE
AFX 1.5.4 - chart functionality broken.

### ISSUE THREAD

**Issue by wfouche** [asciidocfx/AsciidocFX#263]
Hi @rahmanusta,

The chart functionality in AFX 1.5.4 is broken. The text below in a previous version of AFX
```
[chart,bar,file="Key-Roll-Over-CPU.png",opt="legend=bottom, show-labels=true, y-label=CPU Usage (%)", title = "CPU Usage"]
--
//Activity Level 100
CPU (%),25.5
//Activity Level 50
CPU (%),13.2
//Activity Level 10
CPU (%), 3.8
--
```
produces this output.

![image](https://cloud.githubusercontent.com/assets/13682850/20298730/378e2750-ab20-11e6-9570-4e4df9ce45a9.png)

With AFX 1.5.4 we get the following (incorrect) output:

1.  Y-axis numbers are missing.
2.  Background is black, instead of grey.

![image](https://cloud.githubusercontent.com/assets/13682850/20298594/6ddb3ad8-ab1f-11e6-8ec1-5f8e6c2f86d8.png)

Regards,

Werner

**Comment by rahmanusta** [comment]
Could you please test v1.5.5?

**Comment by wfouche** [comment]
Fix confirmed for AFX 1.5.5
